### PR TITLE
Filter emojis out of changelog

### PR DESCRIPTION
**Pull request by AvdLee**
Emojis can break systems like App Store Connect, so it's safer to keep them out by default. I decided to not make it an option or flag since that will require more work now and it's questionable how much it will be used.

### CODE REVIEW

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **ReleaseProducerTests.testSectionedChangelogUpdating():**<br/>XCTAssertEqual failed: ("### 1.0.1
**Closed issues:**

- Include device product names ([#60](https://github.com/WeTransfer/Diagnostics/issues/60))
- Change the order of reported sessions ([#54](https://github.com/WeTransfer/Diagnostics/issues/54))
- Encode Logging for HTML so object descriptions are visible ([#51](https://github.com/WeTransfer/Diagnostics/issues/51))
- Chinese characters display incorrectly in HTML output in Safari ([#48](https://github.com/WeTransfer/Diagnostics/issues/48))
- Get warning for file 'style.css' after building ([#39](https://github.com/WeTransfer/Diagnostics/issues/39))
- Crash happening when there is no space left on the device ([#37](https://github.com/WeTransfer/Diagnostics/issues/37))
- Add support for users without the Apple Mail app ([#36](https://github.com/WeTransfer/Diagnostics/issues/36))
- Support for Apple Watch App Logs ([#33](https://github.com/WeTransfer/Diagnostics/issues/33))
- Support different platforms/APIs ([#30](https://github.com/WeTransfer/Diagnostics/issues/30))
- Strongly typed HTML would be nice ([#6](https://github.com/WeTransfer/Diagnostics/issues/6))

**Merged pull requests:**

- Add charset utf-8 to html head 🫥 ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)

### 1.0.0
- Initial release") is not equal to ("### 1.0.1
**Closed issues:**

- Include device product names ([#60](https://github.com/WeTransfer/Diagnostics/issues/60))
- Change the order of reported sessions ([#54](https://github.com/WeTransfer/Diagnostics/issues/54))
- Encode Logging for HTML so object descriptions are visible ([#51](https://github.com/WeTransfer/Diagnostics/issues/51))
- Chinese characters display incorrectly in HTML output in Safari ([#48](https://github.com/WeTransfer/Diagnostics/issues/48))
- Get warning for file 'style.css' after building ([#39](https://github.com/WeTransfer/Diagnostics/issues/39))
- Crash happening when there is no space left on the device ([#37](https://github.com/WeTransfer/Diagnostics/issues/37))
- Add support for users without the Apple Mail app ([#36](https://github.com/WeTransfer/Diagnostics/issues/36))
- Support for Apple Watch App Logs ([#33](https://github.com/WeTransfer/Diagnostics/issues/33))
- Support different platforms/APIs ([#30](https://github.com/WeTransfer/Diagnostics/issues/30))
- Strongly typed HTML would be nice ([#6](https://github.com/WeTransfer/Diagnostics/issues/6))

**Merged pull requests:**

- Add charset utf-8 to html head ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)

### 1.0.0
- Initial release")
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **ReleaseProducerTests.testPostBodyArguments():**<br/>XCTAssertEqual failed: ("Optional("- Add charset utf-8 to html head 🫥 ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)\n- Get warning for file \'style.css\' after building ([#39](https://github.com/WeTransfer/Diagnostics/issues/39)) via [@AvdLee](https://github.com/AvdLee)")") is not equal to ("Optional("- Add charset utf-8 to html head ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)\n- Get warning for file \'style.css\' after building ([#39](https://github.com/WeTransfer/Diagnostics/issues/39)) via [@AvdLee](https://github.com/AvdLee)")")
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **ReleaseProducerTests.testChangelogUpdating():**<br/>XCTAssertEqual failed: ("### 1.0.1
- Add charset utf-8 to html head 🫥 ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)
- Get warning for file 'style.css' after building ([#39](https://github.com/WeTransfer/Diagnostics/issues/39)) via [@AvdLee](https://github.com/AvdLee)

### 1.0.0
- Initial release") is not equal to ("### 1.0.1
- Add charset utf-8 to html head ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)
- Get warning for file 'style.css' after building ([#39](https://github.com/WeTransfer/Diagnostics/issues/39)) via [@AvdLee](https://github.com/AvdLee)

### 1.0.0
- Initial release")
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **ChangelogItemTests.testUser():**<br/>XCTAssertEqual failed: ("Optional("Add charset utf-8 to html head 🫥 via [@AvdLee](https://github.com/AvdLee)")") is not equal to ("Optional("Add charset utf-8 to html head via [@AvdLee](https://github.com/AvdLee)")")
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **ChangelogItemTests.testAssigneeFallback():**<br/>XCTAssertEqual failed: ("Optional("Add charset utf-8 to html head 🫥 via [@kairadiagne](https://github.com/kairadiagne)")") is not equal to ("Optional("Add charset utf-8 to html head via [@kairadiagne](https://github.com/kairadiagne)")")
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **ChangelogCommandTests.testSectionedChangelogOutput():**<br/>XCTAssertEqual failed: ("- Add charset utf-8 to html head ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)") is not equal to ("- Add charset utf-8 to html head 🫥 ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)")
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **ChangelogCommandTests.testChangelogOutput():**<br/>XCTAssertEqual failed: ("- Add charset utf-8 to html head ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)") is not equal to ("- Add charset utf-8 to html head 🫥 ([#50](https://github.com/WeTransfer/Diagnostics/pull/50)) via [@AvdLee](https://github.com/AvdLee)")
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/44241e52-abdc-4c66-b663-32dc6a658011" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>GitBuddyTests: Executed 71 tests (7 failed, 0 retried, 0 skipped) in 6.187 seconds</td>
    </tr>
  </tbody>
</table>

### SwiftLint found issues

| Severity | File | Reason |
| -------- | ---- | ------ |
Warning | ChangelogItem.swift:38 | Lines should not have trailing whitespace. (`trailing_whitespace`) |
Error | ReleaseCommand.swift:13 | Line should be 160 characters or less: currently 178 characters (`line_length`) |
Error | ReleaseCommand.swift:25 | Line should be 160 characters or less: currently 250 characters (`line_length`) |
Error | ReleaseCommand.swift:29 | Line should be 160 characters or less: currently 185 characters (`line_length`) |
Error | ReleaseCommand.swift:40 | Line should be 160 characters or less: currently 215 characters (`line_length`) |
Warning | PullRequestsJSON.swift:45 | Line should be 140 characters or less: currently 160 characters (`line_length`) |

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |
GitBuddy | 0.00% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against c55572e192960171e425dcc06b89c6053a7b841d
</p>

File: Tests/GitBuddyTests/TestHelpers/PullRequestsJSON.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ let PullRequestsJSON = """
         "number": 50,
         "state": "closed",
         "locked": false,
-        "title": "Add charset utf-8 to html head",
+        "title": "Add charset utf-8 to html head 🫥",
```
Comment (neodym) [inline Tests/GitBuddyTests/TestHelpers/PullRequestsJSON.swift:24]: 🫥❤️

**Review by neodym — APPROVED** [review]
🚢🇮🇹

File: Sources/GitBuddyCore/Commands/ReleaseCommand.swift [hunk 2]
```diff
@@ -21,21 +21,26 @@ struct ReleaseCommand: ParsableCommand {
     @Flag(name: [.customLong("use-pre-release"), .customShort("p")], help: "Create the release as a pre-release.")
     var isPrerelease: Bool = false
 
+    // Main
```
Comment (BasThomas) [inline Sources/GitBuddyCore/Commands/ReleaseCommand.swift:24]: These examples are nice 😊

**Review by BasThomas — APPROVED** [review]

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 4.1.3](https://github.com/WeTransfer/GitBuddy/releases/tag/4.1.3) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**